crates/lib/src/qvm/execution.rs

```diff
@@ -129,11 +129,7 @@ impl Execution {
                 qvm_url: config.qvm_url().into(),
                 source,
             }),
-            Ok(Response::Success(response)) => Ok(response
-                .registers
-                .into_iter()
-                .map(|(key, value)| (key.into_boxed_str(), value))
-                .collect()),
```

Comment:
This transformation turned a `HashMap<String, RegisterData>` into a `HashMap<Box<str>, RegisterData>`. Because the output type was changed, this conversion is no longer necessary.